roles/kube_nginx_ingress/tasks/main.yml

```diff
@@ -1,16 +1,29 @@
+- name: 'create tempfile for nginx-ingress values'
+  tempfile:
+    state: file
+    suffix: yml
+  register: nginx_ingress_value_file
+
+- debug:
+    var: nginx_ingress_value_file.path
+  when: debug|bool
+
+- name: 'copy nginx-ingress values into temporary file'
+  copy:
+    src: nginx_ingress_values.yml
+    dest: '{{ nginx_ingress_value_file.path }}'
+
 - name: 'install nginx ingress'
   command: >-
-    helm install stable/nginx-ingress
-    --name nginx-ingress
+    helm upgrade --install nginx-ingress stable/nginx-ingress
+    --version 0.11.1
```

Comment:
Any chance we can store versions of Helm Charts we deploy in some Ansible `vars` file as well, such that they can be overridden for testing purposes and such?

Reply:
Yes ^^